.eslint-dictionary.json

```diff
@@ -15,13 +16,15 @@
   "amplifyapp",
   "amplifyconfig",
   "amplifyconfiguration",
+  "amplifyconsole",
   "amplifyhosting",
   "amplifymeta",
   "amplifyrc",
   "amplifytools",
   "ansi",
   "apigateway",
   "apigw",
+  "apikey",
```

Comment:
It looks like this is only used in test files. Let's either change spellings or add the eslint-disable flag.

Reply:
removed!